src/main/resources/entity-types/orders/simple_purchase_order_custom_field.json5

```
@@ -0,0 +1,356 @@
+{
+  id: '54695c32-0fe5-4798-ab1b-c66685af2b66',
+  name: 'simple_purchase_order_custom_field',
+  root: false,
+  private: true,
+  sources: [
+    {
+      alias: 'custom_fields',
+      type: 'db',
+      target: 'src_purchase_order_custom_fields',
+    },
+  ],
+  columns: [
+    {
+      name: 'id',
+      sourceAlias: 'custom_fields',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      isIdColumn: true,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ':sourceAlias.id',
```

To make this more readable, you can remove all of the `sourceAlias` properties and change ":sourceAlias" in the `valueGetter`s to ":custom_fields" like this:

```suggestion
      name: 'id',
      dataType: {
        dataType: 'rangedUUIDType',
      },
      isIdColumn: true,
      queryable: true,
      visibleByDefault: false,
      valueGetter: ':custom_fields.id',
```

🙂